dist/components/product/product.js

```diff
@@ -0,0 +1,22 @@
+define(['$'], function ($) {
+
+    return {
+        'product': [
+            {
+                'isHighlight': true,
+                'title': 'Product Title',
+                'url': '#',
+                'image': {
+                    'url': '#'
+                    'path': '#',
```

Comment:
Ah I see. I'm assuming it's the same as `product.url`? In which case `product.image.url` is probably redundant.